shared/lib_resilience.h

```diff
@@ -0,0 +1,166 @@
+#ifndef SYSTEM_ADVISOR_MODEL_LIB_RESILIENCE_H
+#define SYSTEM_ADVISOR_MODEL_LIB_RESILIENCE_H
+
+#include <numeric>
+#include <algorithm>
+#include <functional>
+
+#include "lib_shared_inverter.h"
+#include "lib_battery_dispatch.h"
+#include "lib_battery_powerflow.h"
+#include "../ssc/cmod_battery.h"
```

Comment:
The inclusion of the compute module header causes unresolved externals when building the Test project

Reply:
should be fixed now, please take another look